app/controllers/api/v1/authentication_controller.rb

```diff
@@ -6,13 +6,11 @@ class AuthenticationController < ApiController
       skip_before_action :authenticate, only: :create
 
       def create
-        user = User.find_by_user_name(params.require(:user_name))
-
-        raise ActiveRecord::RecordNotFound, :user_not_found unless user
+        user = User.find_by_user_name!(params.require(:user_name))
 
         raise Errors::AuthenticateError unless user.authenticate(params.require(:password))
 
-        render json: UserSerializer.render_as_json(user, root: :user, view: :with_auth_token),
+        respond_with_success message: UserSerializer.render_as_json(user, root: :user, view: :with_auth_token),
```

Comment:
@snehithlal Should be context, not message.
I also think we shouldnt send the created record, instead just send a notice.

Reply:
Please also check other places we use this.

Reply:
we need auth token and user data. message can be replaced with context